packages/expo-cli/src/commands/install.ts

```diff
@@ -82,23 +82,7 @@ async function installAsync(packages: string[], options: PackageManager.CreateFo
     await packageManager.installAsync();
   }
 
-  const bundledNativeModulesPath = resolveFrom.silent(
-    projectRoot,
-    'expo/bundledNativeModules.json'
-  );
-
-  if (!bundledNativeModulesPath) {
-    Log.addNewLineIfNone();
-    throw new CommandError(
-      `The dependency map ${Log.chalk.bold(
-        `expo/bundledNativeModules.json`
-      )} cannot be found, please ensure you have the package "${Log.chalk
-        .bold`expo`}" installed in your project.\n`
-    );
-  }
-
-  const bundledNativeModules = await JsonFile.readAsync(bundledNativeModulesPath);
-
+  const bundledNativeModules = await getBundledNativeModulesAsync(projectRoot, exp.sdkVersion);
```

Comment:
Is it possible for `exp.sdkVersion` to be undefined? If so does the endpoint account for that? Also what happens during EXPO_BETA or unversioned?

Reply:
@dsokal - that is a correct understanding of beta versions, yes!